docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+
+services:
+  db:
+    image: postgres:13-alpine
+    container_name: example-db
```

Comment:
Mandy suggested changing container_name. It'll prevent any db conflicts when others run locally. Leia added to run `docker compose up ` on the command line after, push the changes, and that should do it.